File: solutions/0001-two-sum/solution.py
"""
LeetCode: [0001] Two Sum
Link: https://leetcode.com/problems/two-sum/
Date: 2026-06-01
Language: python
Status: Accepted
"""

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        for i, x in enumerate(nums):
            for j, y in enumerate(nums):
                if (i != j) and x + y == target:
                    return i, j


if __name__ == "__main__":
    s = Solution()
    result = s.twoSum([2, 7, 11, 15], 9)
    print(result)

File: solutions/0001-two-sum/solution.js
/**
 * LeetCode: [0001] Two Sum
 * Link: https://leetcode.com/problems/two-sum/
 * Date: 2026-06-01
 * Language: javascript
 * Status: Accepted
 */

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (i = 0; i < nums.length - 1; i++) {
        for (j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] === target)
                return [i, j];
        }
    }
};